style.css

```diff
@@ -0,0 +1,499 @@
+  border-radius: 4px;
+}
+
+.project-button {
+  font-family: 'Inter', sans-serif;
+  background-color: #36b37e;
+  border: none;
+  color: white;
+  padding: 12px;
+  text-align: center;
+  text-decoration: none;
+  font-size: 17px;
+  border-radius: 4px;
+  margin-left: 110px;
+  margin-top: 40px;
+  cursor: pointer;
+}
+
+.about-me-title {
+  display: flex;
+  justify-content: flex-end;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  height: 100px;
+}
+
+.about-me-detail {
+  width: 351px;
+  height: 189px;
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: center;
+  margin-left: 70px;
+}
+
+#about-me-text {
+  font-size: 40px;
+  padding-right: 90px;
+}
+
+.standard-button {
+  background-color: #36b37e;
+  border: none;
+  color: black;
+  padding: 12px;
+  text-align: center;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  border-radius: 4px;
+  cursor: pointer;
+  margin-top: 15px;
+  margin-left: 180px;
+}
+
+.education {
+  display: grid;
+  row-gap: 20px;
+}
+
+.education-title {
+  font-family: 'Inter', sans-serif;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+}
+
+#education-ul-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+#education-ul-list li {
+  display: inline;
+  background-color: #fff;
+  color: #36b37e;
+  margin: 2px;
+  padding: 8px;
+  border-radius: 8px;
+}
+
+.education-list-2 {
+  padding-top: 20px;
+  padding-right: 50px;
+}
+
+#education-ul-list-2 li {
+  display: inline;
+  background-color: #fff;
+  color: #36b37e;
+  margin: 2px;
+  padding: 12px;
+  border-radius: 8px;
+}
+
+.about_me_section {
+  display: grid;
+  row-gap: 20px;
+  grid-template-rows: auto;
+}
+
+.contact-form-section {
+  display: flex;
+  flex-direction: column;
+}
+
+#contact-form-desc {
+  width: 327px;
+  height: 320px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  padding-left: 18%;
+  margin-bottom: 100px;
+}
+
+.form {
+  margin-left: 17%;
+}
+
+input {
+  margin-bottom: 30px;
+}
+
+#fullname,
+#email {
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+}
+
+#textarea {
+  width: 327px;
+  height: 114px;
+  border-radius: 4px;
+  border-color: #60c095;
+}
+
+#get-in-btn {
+  margin-top: 20px;
+  margin-left: 110px;
+}
+
+#social-media-links-2 {
+  margin-left: 24%;
+  margin-top: 10px;
+}
+
+#footer-email {
+  width: 375px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  color: #42526e;
+  margin-left: 25px;
+}
+
+.green-button:hover,
+.project-button:hover {
+  background-color: #14e7dd;
+  color: black;
+}
+
+.green-button:active,
+.project-button:active {
+  background-color: #14e7dd;
+  color: black;
+  box-shadow: 0 5px #666;
+  transform: translateY(4px);
+}
+
```

Comment:
- [ ] Kindly make sure the hover and active state of your project matches the design template. You are expected to stick to design guidelines as much as possible.
###Figma template
![f4](https://user-images.githubusercontent.com/94551484/194693552-b9bfb924-492c-4ed2-97c2-b1f7cabddf8d.png)